mlflow/azureml/__init__.py

```diff
@@ -38,17 +38,17 @@ def build_image(model_uri, workspace, image_name=None, model_name=None,
     For information about the input data formats accepted by this webserver, see the
     :ref:`MLflow deployment tools documentation <azureml_deployment>`.
 
-    :param model_uri: The location, in URI format, of the MLflow model for which to build an Azure
-                      ML deployment image, for example:
+    :param model_uri: The location, in URI format, of the MLflow model to build an Azure
+                      ML deployment image of. For example:
```

Comment:
I think this sentence is confusing due to the ending preposition. Can we revert it or try something else?

Reply:
@stbof can you address this?